docs/source/primitives.rst

```diff
@@ -50,15 +50,15 @@ Returning values from the context scope to the flow above is done by setting the
 
 .. note:: The ``result`` object should copied then updated by parsing the ``result_keys`` this allows meta-data/debug-info to be set by the provider of the workflow.
 
-The `result` is either placed at ``destination_path`` or if ``destination_path = False`` the ``result`` is merged directly into the context above ASK_MICKY (parent context?).
+The `result` is either placed at ``destination_path`` or if ``destination_path = False`` the ``result`` is merged directly into the parent context, that is the context of the flow/task of which this is a subordinate. ASK_MICKY (have i made it better or worse?)
```

Comment:
only flow tasks have tasks - redraft with that in mind. Subordinate is confusing, subtask as used elsewhere probably